notebooks/drive_anon_example.ipynb

```diff
@@ -116,35 +113,16 @@
     "Create a data directory, use driveanon to save the data to it, and unzip."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "! mkdir ../data"
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": 6,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[Errno 2] No such file or directory: '../data'\n",
-      "/Users/knuth/Desktop/ice_sat_hack/topohack/notebooks\n",
-      "/Users/knuth/Desktop/ice_sat_hack/topohack/notebooks\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# this might take a little bit to download the 6 GB of data\n",
-    "%cd ../data\n",
+    "%cd ../../data\n",
     "da.save(blob_id)\n",
-    "%cd ../notebooks"
+    "%cd ../topohack/notebooks"
```

Comment:
Shouldn't this stay at `../notebooks` since we will have a `data/` directory under the git repo?

Reply:
I think we want data/ at our ~/ home directory, instead of inside the topohack github repo. This was suggested by @willkochtitzky today and makes sense to me. Another guard against accidentally committing data to GitHub, in addition to adding that folder to the gitignore.

Reply:
The magic command %cd wasn't playing nice with $HOME but it looks like it works with ~/ Will change to used ~/ instead of a relative path.